src/higlass/types.ts

```diff
@@ -0,0 +1,76 @@
+// HiGlass's plugin system goes through globals
```

Comment:
HiGlass doesn't provide any types. I've tried to type out the interfaces that we require so we can appropriately type-check the file and also help with autocomplete, etc.